src/main/scala/eventstore/Settings.scala

```diff
@@ -84,4 +87,17 @@ object Settings {
    * Java API
    */
   def getInstance(): Settings = Default
-}
\ No newline at end of file
+}
+
+case class HttpSettings(protocol: String = "http", port: Int = 2113, prefix: String = "", url: String = "http://127.0.0.1:2113/") {
```

Comment:
I think it should look like `case class HttpSettings(uri: Uri = Uri("http://127.0.0.1:2113"))`